packages/block-editor/src/components/block-tools/insertion-point.js

```diff
@@ -81,36 +79,9 @@ function InbetweenInsertionPointPopover( {
 			isInserterShown: insertionPoint?.__unstableWithInserter,
 		};
 	}, [] );
-	const { getBlockEditingMode } = useSelect( blockEditorStore );
 
 	const disableMotion = useReducedMotion();
 
-	function onClick( event ) {
-		if (
-			event.target === ref.current &&
-			nextClientId &&
-			getBlockEditingMode( nextClientId ) !== 'disabled'
-		) {
-			selectBlock( nextClientId, -1 );
-		}
-	}
-
-	function maybeHideInserterPoint( event ) {
-		// Only hide the inserter if it's triggered on the wrapper,
-		// and the inserter is not open.
-		if ( event.target === ref.current && ! openRef.current ) {
-			hideInsertionPoint();
```

Comment:
These things seem to be workarounds to the fact that the indicator caught clicks (and didn't trigger the `mousemove` event listener in the canvas, so it had to be duplicated here.